splash.html

```diff
@@ -1,22 +1,126 @@
 <!DOCTYPE html>
 <html lang="en">
+
+<!--PAGE SETUP-->
 <head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link href="css/normalize.css" rel="stylesheet" type="text/css">
-    <link href="css/splash.css" rel="stylesheet" type="text/css">
-    <link href="https://fonts.googleapis.com/css?family=Permanent+Marker" rel="stylesheet">
-    <title>whack-a-rat!</title>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>Whack-a-Rat</title>
+    <meta name="description" content="Mayden Academy JS Game">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="css/normalize.css">
+    <link rel="stylesheet" href="css/game.css">
 </head>
-<body class="body">
+
+<!--SPLASH SCREEN-->
+
+<div class="splash">
     <div class="gamescreen">
         <img class="image" src="images/exploding.svg">
```

Comment:
there shouldn't be two images folder. Move the exploding img to the assets folder and delete the duplicated 'images' folder in the root dir